### ISSUE TITLE
[F] Preview table as HTML

### ISSUE THREAD

**Issue by Akxe** [exceljs/exceljs#1603]
## 🚀 Feature Proposal

I would like to be able to preview the whole or part of a sheet in HTML.

## Motivation / Example

You have an export button on your page and you want the user to be able to quickly preview part of the table before the export.

This would also really help with debugging table creations and would enable for awesome examples in docs pages.

-----

I would say this could be created as an "in-preview" feature, while only implementing some parts. The most common parts should be:

* Font
  * Font family
  * Font-size
  * Font style (bold, italics)
* Background colour (solid only)
* Foreground colour (text colour)
* Cells borders
* Merged cells

As for implementation, I think the output should be a `<div>` with `<style>` and `<table>` inside. The style would contain one class per every unique style object in the sheet.

```html
<div>
  <style>
  #excelJS-123 td {
    /** default TD style */
    border: ...;
    ...
  }
  #excelJS-123 td.excelJS-td-1 { ... }
  #excelJS-123 td.excelJS-td-2 { ... }
  </style>
  <table id="excelJS-123">
    <tr>
      <td></td>
      <td class="excelJS-td-1"></td>
      <td class="excelJS-td-2"></td>
    </tr>
  </table>
</div>
```

The div is created to clump styles with the table together. Removing the `<div>` from the document will delete the table along with its styles.

**Comment by nslyc** [comment]
Yeah, I also hope that it has the ability to convert to HTML format while preserving the style.
